proxy-node-shared/src/main/java/uk/gov/ida/notification/exceptions/mappers/VerifyServiceProviderGenerateAuthnRequestResponseExceptionMapper.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.ida.notification.exceptions.mappers;
+
+import uk.gov.ida.notification.exceptions.proxy.VerifyServiceProviderGenerateAuthnRequestResponseException;
+import uk.gov.ida.notification.views.ErrorPageView;
+
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import java.util.logging.Logger;
+
+public class VerifyServiceProviderGenerateAuthnRequestResponseExceptionMapper implements ExceptionMapper<VerifyServiceProviderGenerateAuthnRequestResponseException> {
```

Comment:
Maybe rename this to `VspGenerateAuthnRequestResponseExceptionMapper` to make it a manageable length? 😅